pylibczi/CziFile.py

```diff
@@ -186,10 +268,13 @@ def read_image(self, m_index: int = -1, **kwargs):
                        H = 7   # The H-dimension ("phase").
                        V = 8   # The V-dimension ("view").
 
-        :returns: a tuple of (numpy.ndarray, a list of (Dimension, size)) the second element of the tuple is to make
+        Returns
+        -------
+        (numpy.ndarray, [Dimension, Size])
+            a tuple of (numpy.ndarray, a list of (Dimension, size)) the second element of the tuple is to make
             sure the numpy.ndarray is interpretable. An example of the list is ::
```

Comment:
I think you forgot to remove the `::` at the end of this when switching to numpy format.